### ISSUE TITLE
'backward' direction scan?

### ISSUE THREAD

**Issue by gfdsah236** [TerryPei/EfficientVMamba#5]
It seems that your 4 directional scans do not have the backward directions.  Why?

**Comment by TerryPei** [comment]
Yes. Reason: in the early stages, we tried both horizontal and vertical forward scanning as well as reverse scanning, consistent with Vmamba's methodology. Yet, the results were less competitive compared to current design.

**Comment by TerryPei** [comment]
> It seems that your 4 directional scans do not have the backward directions. Why?

@gfdsah236 Yes. Reason: in the early stages, we tried both horizontal and vertical forward scanning as well as reverse scanning, consistent with Vmamba's methodology. Yet, the results were less competitive compared to current design.

**Comment by gfdsah236** [comment]
Thanks for your reply. Where can I find the comparison results?
